ci_build/outer_launch_tests.sh

```diff
@@ -8,5 +8,6 @@ STACK_RESOLVER=${STACK_RESOLVER:-lts-6.2}
 IMAGE_NAME=tensorflow/haskell/ci_build:$STACK_RESOLVER
 
 git submodule update
+docker pull tensorflow/tensorflow:nightly-devel
```

Comment:
I think 0.12.1-devel is not compatible with recent TF. We'll probably have to wait until newer tag before pinning it.